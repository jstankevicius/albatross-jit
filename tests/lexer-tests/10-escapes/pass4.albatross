# things can get interesting
"how many escapes here? \\\\\\\\\\\\"
"how about here? \\\n\\\\t\\\n\\"
