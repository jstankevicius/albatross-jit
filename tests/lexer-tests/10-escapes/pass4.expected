1 2 STRING 29 how many escapes here? \\\\\\
1 3 STRING 24 how about here? \
\\t\
\
